contracts/AmplificationUtils.sol

```diff
@@ -22,6 +22,65 @@ library AmplificationUtils {
     uint256 private constant MAX_A_CHANGE = 2;
     uint256 private constant MIN_RAMP_TIME = 14 days;
 
+    /**
```

Comment:
Any specific rationale on using solidity 0.6.12?

Reply:
A lot of this code is copied and refactored from `Swap.sol` so I wanted to keep the changes to minimum. 
Though nowadays I see a lot of projects embrace 0.8.* ... 🤔

Reply:
I've seen a 20% gas improvement after upgrading. But it may be bring in a lot of changes so be careful.